Gemfile.lock

```diff
@@ -319,4 +320,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.14.4
+   1.14.6
```

Comment:
Something I've been seeing (with dep versions shifting ahead of us here) are loading issues with the version of bundler. Because of the CI setup, it'll end up loading multiple versions and then fails. What do you think of reverting this line on the assumption that we'll re-run before the next release?